modules/controller/landing_pack_tracking.py

```diff
@@ -0,0 +1,91 @@
+"""
+keeps track of detected and visited landing pads
+"""
+
+import numpy as np
+
+from .. import object_in_world  #Edit name when finalized
+
+
+class LandingPadTracking:
+    """
+    keeps track of the real world location of landing pads that
+    are to be visited or have already been vsisited
+    """
+    def __init__(self, distance_threshold: float):
+        self.__unconfirmed_positives = []
+        self.__false_positives = []
+        self.__confirmed_positives = []
+
+        # Landing pads within the square root of this distance are considered the same landing pad
+        self.__distance_threshold = distance_threshold
+
+    @staticmethod
+    def __similar(detection1: object_in_world.ObjectInWorld, detection2: object_in_world.ObjectInWorld, distance_squared_threshold: float) -> bool:
+        """
+        returns whether detection1 and detection2 are close enough
+        to be considered the same landing pad
+        """
+
+        distance_squared = (detection2.position_x - detection1.position_x) ** 2 + (detection2.position_y - detection1.position_y) ** 2
+        return distance_squared < distance_squared_threshold
+
+    def mark_false_positive(self, detection: object_in_world.ObjectInWorld):
+        """
+        marks a detection as false positive and removes similar landing
+        pads from the list of unconfirmed positives
+        """
+
+        self.__false_positives.append(detection)
+        for landing_pad in self.__unconfirmed_positives:
+            if self.__similar(landing_pad, detection, self.__distance_threshold):
+                self.__unconfirmed_positives.remove(landing_pad)
+
+    def mark_confirmed_positive(self, detection: object_in_world.ObjectInWorld) -> object_in_world.ObjectInWorld:
+        """
+        marks a detection as a confimred positive for future use
+        """
+
+        self.__confirmed_positives.append(detection)
+        
+    def run(self, detections: np.ndarray):
```

Comment:
Type is `list[object_in_world.ObjectInWorld]` . Put in inside double quotation marks as well since Python 3.8 type annotations doesn't support containers without importing the typing module and we want to avoid importing the typing module.